Gems/EMotionFX/Code/Include/Integration/ApplyMotionSetComponentBus.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#if defined(CARBONATED)
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Component/ComponentBus.h>
+#include <AzCore/Math/Vector2.h>
+#include <AzCore/Math/Vector3.h>
+#include <AzCore/Math/Quaternion.h>
+#include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Component/EntityId.h>
+
+namespace EMotionFX
+{
+    namespace Integration
+    {
+#if defined(CARBONATED)

```

Comment:
Looks like inner #ifdef/#endif  for CARBONATED are not necessary due to we use #if defined(CARBONATED) for whole file in the line 3
?

Reply:
I just wanted to keep it closer to the LY source. If you wish I can refactor it.